monai/data/image_reader.py

```diff
@@ -924,7 +952,7 @@ def read(self, data: Sequence[PathLike] | PathLike, **kwargs):
             img_.append(img)  # type: ignore
         return img_ if len(filenames) > 1 else img_[0]
 
-    def get_data(self, img) -> tuple[np.ndarray, dict]:
+    def get_data(self, img) -> tuple[np.ndarray | cp.ndarray, dict]:
```

Comment:
Does the typing work if CuPy isn't installed? I don't know if our CICD tests are going to catch when it's not.

Reply:
Thanks @ericspod . TYPE_CHECKING requires the library is installed, but install cupy and kvikio in our CI/CD machine seems not easy (cannot simply add in requirements.txt)
Therefore, I remove this type hint and remove the import from TYPE_CHECKING. Do you think it's fine?

Reply:
Thank, I added a TODO in the code